terraform/eks/variables.tf

```diff
@@ -0,0 +1,68 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+variable "region" {
+  description = "AWS region"
+  type        = string
+  default     = "us-east-2"
+}
+
+variable "runners_node_types" {
+    description = "Node type for the runners"
+    type        = list(string)
+    default     = [
+      "t4g",
+      "m7g",
+      "m6g",
+    ]
+}
+
+variable "x64_runners_node_types" {
+  description = "X64 node type for the runners"
+  type        = list(string)
+  default     = [
+    "m7a",
+    "m6a",
+    "t3",
+    "t3a",
+  ]
+}
+
+variable "small_runners_node_size" {
+    description = "Node size for the small runners"
+    type        = string
+    default     = "medium"
+}
+
+variable "small_runners_x64_node_size" {
+  # This is a separate variable because the r series doesn't have a medium size
+  description = "Node size for the small runners"
+  type        = string
+  default     = "large"
+}
+
+variable "medium_runners_node_size" {
+    description = "Node size for the medium runners"
+    type        = string
+    default     = "xlarge"
+}
+
+variable "large_runners_node_size" {
+    description = "Node size for the large runners"
+    type        = string
+    default     = "2xlarge"
+}
```

Comment:
Nit: I'm not actually sure having this as variables makes it clearer -- to be (and this just a nit) I find this variable use makes it harder to understand)

```hcl
      instance_types = [for node_type in var.runners_node_types: "${node_type}.${var.large_runners_node_size}"]
```

vs 

```hcl
      instance_types = [for node_type in var.runners_node_types: "${node_type}.large"]
```

Reply:
My goal was to avoid touching the code to change these configurations, as we may need to change it according to the performance of our CI jobs. I will try to simplify it.